library/src/options.cxx

```diff
@@ -216,6 +216,9 @@ options::options()
   this->Internals->init("model.volume.enable", false);
   this->Internals->init("model.volume.inverse", false);
 
+  // Camera projection
+  this->Internals->init("camera.orthographic", false);
```

Comment:
doc for this should be added in doc/libf3d/OPTIONS.md

Reply:
It looks like the `OPTIONS.md` is for command line options. 
This feature is not a command line option at the moment.